net/ghttp/ghttp.go

```diff
@@ -12,6 +12,8 @@ import (
 	"reflect"
 	"time"
 
+	"github.com/gogf/gf/v2/errors/gcode"
+	"github.com/gogf/gf/v2/errors/gerror"
```

Comment:
gofmt

Reply:
It's already `gofmt -w`, it or else could not go through CU procedure.